### PR TITLE
Nikolay/fix/password confirm

### PR DESCRIPTION
**Pull request by kipod**

### CODE REVIEW

File: app/forms/auth.py [hunk 1]
```diff
@@ -41,3 +41,11 @@ def validate_username(form, field):
...
         if User.query.filter_by(email=field.data).first() is not None:
             raise ValidationError("This email is already registered.")
+
+
+class EditUserForm(RegistrationForm):
+    def validate_username(form, field):
+        pass
+
+    def validate_email(form, field):
+        pass
```
Comment (Ziles131) [inline app/forms/auth.py:52]:
```
class EditUserForm(RegistrationForm):
    def validate_username(form, field):
        pass

    def validate_email(form, field):
        pass
```
Is this code should be empty?

File: app/views/team.py [hunk 2]
```diff
@@ -1,23 +1,32 @@
...
 def edit_card(user_id):
-    form = RegistrationForm(request.form)
-    user = User.query.get(user_id)
-    user.username = form.username.data
-    user.email = form.email.data
-    user.position = form.position.data
-    user.phone = form.phone.data
-    user.password = form.password.data
-    user.save()
+    form = EditUserForm(request.form)
```
Comment (Ziles131) [inline app/views/team.py]: I think "request.form" set as a default value in "EditUserForm" class

File: app/forms/auth.py (hunk 1 above)
Comment (kipod) [reply app/forms/auth.py:52]: I wanted to said "Yes", but found the bug in this logic... Thank you!

File: app/views/team.py (hunk 2 above)
Comment (kipod) [reply app/views/team.py]: yes

**Pull request merged**